custom/icds/templates/location_rationalization/request.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'hqwebapp/two_column.html' %}
```

Comment:
This should extend `base_section.html` to get breadcrumbs.

Reply:
https://github.com/dimagi/commcare-hq/pull/26865/commits/33c80236959e64d07120378b8a9475c16a665579